TestHelpers/utils.sh

```diff
@@ -25,12 +25,8 @@ development_clevis() {
     else
         rlRun "dnf install python-pip gcc clang cmake jose libjose-devel cryptsetup-devel socat tpm2-tools luksmeta libluksmeta-devel -y"
     fi
-
```

Comment:
We will need a couple more changes here to make it work. 

- remove this line here - https://github.com/RedHat-SP-Security/clevis-tests/blob/master/Sanity/pkcs11/basic/main.fmf#L6. It is installing clevis early during some phase controlled by tmt, and it then breaks this functionality. For some reason it installs some old version `clevis-20-2.fc40.x86_64.rpm` which does not contain the new features. You will need to remove this in all the `main.fmf` files in [these](https://github.com/RedHat-SP-Security/clevis-tests/tree/master/Sanity/pkcs11) directories as well. 
- you will need to move [this](https://github.com/RedHat-SP-Security/clevis-tests/blob/master/Sanity/pkcs11/basic/runtest.sh#L58) function call in all the tests to the top, as a first thing (before [this](https://github.com/RedHat-SP-Security/clevis-tests/blob/master/Sanity/pkcs11/basic/runtest.sh#L37) line). With that I also suggest renaming the function to something like `install_clevis()` and remove the installation of the required packages by meson - https://github.com/RedHat-SP-Security/clevis-tests/blob/master/TestHelpers/utils.sh#L24. 
- One last question, is the new `clevis-pin-pkcs11` package available also for Fedora39? Because as of now, we are still running the tests on that distro (since it is still officially supported released fedora version). If not, we can stop running it there. Also, we should add the RHEL-9.6 and RHEL-10 to the mix.

Reply:
> We will need a couple more changes here to make it work.
> 
>     * remove this line here - https://github.com/RedHat-SP-Security/clevis-tests/blob/master/Sanity/pkcs11/basic/main.fmf#L6. It is installing clevis early during some phase controlled by tmt, and it then breaks this functionality. For some reason it installs some old version `clevis-20-2.fc40.x86_64.rpm` which does not contain the new features. You will need to remove this in all the `main.fmf` files in [these](https://github.com/RedHat-SP-Security/clevis-tests/tree/master/Sanity/pkcs11) directories as well.
> 
>     * you will need to move [this](https://github.com/RedHat-SP-Security/clevis-tests/blob/master/Sanity/pkcs11/basic/runtest.sh#L58) function call in all the tests to the top, as a first thing (before [this](https://github.com/RedHat-SP-Security/clevis-tests/blob/master/Sanity/pkcs11/basic/runtest.sh#L37) line). With that I also suggest renaming the function to something like `install_clevis()` and remove the installation of the required packages by meson - https://github.com/RedHat-SP-Security/clevis-tests/blob/master/TestHelpers/utils.sh#L24.
> 
>     * One last question, is the new `clevis-pin-pkcs11` package available also for Fedora39? Because as of now, we are still running the tests on that distro (since it is still officially supported released fedora version). If not, we can stop running it there. Also, we should add the RHEL-9.6 and RHEL-10 to the mix.

Regarding `clevis-pin-pkcs11`, it has been added to Fedora-41 and Fedora-40, not Fedora-39. I can confirm it is present also in latest RHEL9 and RHEL10 composes (20241106)